pubspec.yaml

```diff
@@ -42,6 +42,7 @@ dependencies:
   firebase_core: ^0.4.3+1
   firebase_analytics: ^5.0.9
   flutter_staggered_grid_view: ^0.3.0
+  crop: any
```

Comment:
Please use fixed / latest version of the package

Reply:
I did both changes. Thanks.